composer.json

```diff
@@ -137,6 +140,11 @@
         "drupal/entity_browser": "8.1.0-alpha3",
         "drupal/views_infinite_scroll": "8.1.0",
         "drupal/media_entity_document": "^8.1.0",
-        "drupal/video_embed_field": "^8.1.0"
+        "drupal/video_embed_field": "^8.1.0",
+        "fxp/composer-asset-plugin": "dev-master#b2cc94c8018a7f4d61d1e0ab0d211a2713daa4fe",
```

Comment:
Composer doesn't let you pull specific commit hashes for dependencies of dependencies. So when lightning is a dependency of BLT or Lightning Project, this won't work.

Can you pull 1.1.4 and patch whatever functionality is missing?